geomesa-hbase/geomesa-hbase-native/src/main/scala/org/locationtech/geomesa/hbase/nativeapi/HBaseGeoIndex.scala

```diff
@@ -0,0 +1,93 @@
+/** *********************************************************************
+* Copyright (c) 2013-2017 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+* ************************************************************************/
+package org.locationtech.geomesa.hbase.nativeapi
+
+import org.apache.hadoop.classification.InterfaceStability
+import org.apache.hadoop.hbase.client.Connection
+import org.geotools.data.DataStoreFinder
+import org.locationtech.geomesa.api._
+import org.locationtech.geomesa.hbase.data.{HBaseDataStore, HBaseDataStoreParams, HBaseFeatureWriter}
+
+import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
+
+@InterfaceStability.Unstable
+class HBaseGeoIndex[T](override protected val ds: HBaseDataStore,
```

Comment:
For consistency with accumulo, should probably be `HBaseGeoMesaIndex` (or change accumulo one)